locale/en/docs/guides/diagnostics-flamegraph.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Diagnostics - Flame Graphs
+layout: docs.hbs
+---
+
+# Flame Graphs
+
+## What's a flame graph useful for?
+
+Flame graphs are a way of visualizing CPU time spent in functions. They can help you pin down where you spend too much time doing synchronous operations.
+
+## How to create a flame graph
+
+You might have heard creating a flame graph for Node.js is difficult, but that's not true (anymore).
+Solaris vms are no longer needed for flame graphs on linux!
+
+1. install lldb in your system (eg. with apt-get)
+1. install llnode according to readme in [github project](https://github.com/nodejs/llnode)
+1. install linux-tools-common (eg. with apt-get)
+1. try running `perf` - it might complain about missing kernel modules, install them too
+1. install stackvis `npm i -g stackvis`
+1. run node with perf enabled `perf record -e cycles:u -g -- node --perf-basic-prof app.js`
```

Comment:
I guess the number at the beginning reflect a step in the process. If so they should be incremented at every step.

Reply:
It's how markdown works. Generated HTML will have an ordered list here.
I could switch to incremental so the MD file is more readable, but it's not meant to be read without rendering.

Reply:
I didn't know 😮. always used `1. 2. 3. ...`. Thanks.